app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -357,7 +391,10 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
     }
   }
 
-  function getPoolRecommendation(poolHash: string, stake: number): Promise<any> {
+  function getPoolRecommendation(
+    poolHash: string,
+    stake: number
```

Comment:
do you know what's this "stake" parameter? seems pretty cryptic to me, so maybe it would be nice to give it some more descriptive name